api/src/main/java/cz/muni/fi/pa165/facade/UserFacade.java

```diff
@@ -68,6 +68,12 @@
      * */
     List<StudyGroupDTO> findUserStudyGroups(Long userID);
 
+    /**
+     * Find Study groups which is user regstered as a leader in
+     * @param userID ID of the user
+     * */
+    List<StudyGroupDTO> findLeadersStudyGroups(Long userID);
```

Comment:
This method name makes no sense. Better doc -  "Find Study groups lead by the user".